app/boards/shields/a_dux/a_dux.zmk.yml

```
@@ -0,0 +1,11 @@
+file_format: "1"
+id: a_dux
+name: A. Dux
+type: shield
+url: https://github.com/tapioki/cephalopoda/
```

Can we actually use the more direct URL here as well, same one from the README file:

```suggestion
url: https://github.com/tapioki/cephalopoda/
```
```suggestion
url: https://github.com/tapioki/cephalopoda/tree/main/Architeuthis%20dux
```

---

Absolutely! I've just updated the PR with this change to the URL. :+1: